packages/react-app/src/components/GiveawayCard.tsx

```diff
@@ -1,35 +1,55 @@
-import { Card, CardMedia, CardContent, Typography, CardActions, Button } from "@mui/material";
+import { Card, CardMedia, CardContent, Typography, Box, Button, Skeleton } from "@mui/material";
 import { Giveaway } from "../lib/types";
 import { format } from "date-fns";
+import { useNavigate } from "react-router-dom";
 
 const GiveawayCard = (props: Giveaway) => {
+  const navigate = useNavigate();
+
   return (
-    <Card>
+    <Card sx={{ borderRadius: '1.2rem' }}>
       <CardMedia
         component="img"
-        height="140"
-        // image={props.image ? `${config.API}/${props.image}` : '/static/images/cards/contemplative-reptile.jpg'}
+        height="120"
+        image={props.image ? props.image : '/static/images/placeholder-image.jpg'}
       />
       <CardContent>
-        <Typography gutterBottom variant="h5" component="div">
+        <Typography gutterBottom variant="h5">
           {props.title}
         </Typography>
-        <Typography variant="body2" color="text.secondary">
+        <Typography gutterBottom>
           {props.description}
         </Typography>
-        <Typography variant="body2" color="text.secondary">
-          {props.prize}
+        <Typography gutterBottom>
+          <>🏆 {props.prize}</>
```

Comment:
Fragments are not needed inside components, only as wrappers
```suggestion
          🏆 {props.prize}
```